integreat_cms/cms/forms/translations/translations_management_form.py

```diff
@@ -26,3 +76,52 @@ class Meta:
             "machine_translate_events",
             "machine_translate_pois",
         ]
+
+    def __init__(self, *args, **kwargs):
+        r"""
+        Initialize translations management form
+
+        :param \*args: The supplied arguments
+        :type \*args: list
+
+        :param \**kwargs: The supplied keyword arguments
+        :type \**kwargs: dict
+        """
+        super().__init__(*args, **kwargs)
+
+        self.fields[
+            "machine_translatable_languages"
+        ].queryset = self.instance.language_tree_nodes.filter(active=True).order_by(
+            "language"
+        )
```

Comment:
I think it would be cool to group the languages by MT provider instead of ordering them by the language:

![Screenshot 2023-02-07 at 13-15-03 Integreat Editorial System](https://user-images.githubusercontent.com/16021269/217242710-123f090f-c467-4b77-b1c0-070f23f972be.png)

In order to do so, we need to explicitly set the choices of the field in addition to the queryset:

```suggestion
        # Exclude inactive languages and the root node
        languages = self.instance.language_tree_nodes.filter(
            active=True, parent__isnull=False
        )
        mt_providers = defaultdict(list)
        self.unavailable_languages = []
        for language in languages:
            if language.mt_provider:
                mt_providers[language.mt_provider].append(
                    (language.id, language.translated_name)
                )
            else:
                self.unavailable_languages.append(language.translated_name)
        self.fields["machine_translatable_languages"].choices = mt_providers.items()
        self.fields["machine_translatable_languages"].queryset = languages.filter(
            id__in=[id for languages in mt_providers.values() for id, _ in languages]
        )
```